### DISCUSSION TITLE
HIP idea Trash wallet

### DISCUSSION THREAD

**Discussion by firedawg225** [hiero-ledger/hiero-improvement-proposals#536] [Ideas]
Creating a trash wallet or a way to people to get rid of unwanted NFT's out of there wallets. Could be rugged projects, ghosted developers, or whatever. No token association just a place to send unwanted NFTs. Sol has Sol.trash as the address and you can send whatever you want there.

**Comment by bmgentile** [comment]
+1 am creating a HIP for this right now — if you'd like to contribute here is the draft: https://docs.google.com/document/d/12L5uEynwxEhPvJ5FIGIrrnn9VgCX1Mf5-srp0EE74bQ/edit?pli=1

We looked into the idea of a "Burn Account" that tokens can be sent to, but realized it would clog up storage and is limited to 1000 token IDs (limitation on Hedera wrt "token slots"); the conclusion was a "Token Destroy" transaction that can be signed by any account holding an NFT or fungible token(s) of any kind, without needing sign-off by an admin key (or if there is no admin key).

This also solves the problem incurred by using a "burn account", whereby once the burn account expires, tokens end up going back to their treasury — if it's the treasury account itself trying to get rid of the tokens, it would just end up going back to them once the burn account expired.
